src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/DeleteDraftApprenticeshipRequestMapper.cs

```diff
@@ -0,0 +1,15 @@
+﻿using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.ProviderCommitments.Web.Models.Apprentice;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ProviderCommitments.Web.Mappers.Apprentice
+{
+    public class DeleteDraftApprenticeshipRequestMapper : IMapper<DeleteConfirmationViewModel, DeleteDraftApprenticeshipRequest>
```

Comment:
is this mapper class necessary. I know it is the convention but in this instance seems a little pointless

Reply:
need the new instance and this way can unit test

Reply:
could just create the new instance in the controller action and unit test would work fine as there is nothing being set so no mocking would be necessary?

Reply:
could just create the new instance in the controller action and unit test would work fine as there is nothing being set so no mocking would be necessary - done